csharp/autobuilder/Semmle.Autobuild.Cpp/Semmle.Autobuild.Cpp.csproj

```diff
@@ -1,19 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
-    <AssemblyName>Semmle.Autobuild.Cpp</AssemblyName>
-    <RootNamespace>Semmle.Autobuild.Cpp</RootNamespace>
```

Comment:
Not needed (the same goes for some of the other removed properties)?

Reply:
What do you mean by "Not needed"? The linked line is removed in the PR.

Reply:
I see. I think nothing changes when we remove these properties. I think all of them were set to the default values, which would be the name of the project.